src/dotnet/README.md

```diff
@@ -65,7 +65,7 @@ The following commands exist for dotnet.
 
 `dotnet new`
 
-    Initializes a sample .NET Core console application that can be compiled and ran.
+    Initializes a sample .NET Core console or ASP.NET Core web application that can be compiled and run.
```

Comment:
This statement should really be left generic and not be something where every template we add get listed in the description.